src/modules/DigiTrustCommunication.js

```diff
@@ -104,6 +104,16 @@ DigiTrustCommunication._messageHandler = function (evt) {
             case 'DigiTrust.setAppsPreferences.response':
                 helpers.MinPubSub.publish('DigiTrust.pubsub.app.setAppsPreferences.response', [evt.data.value]);
                 break;
+            case 'Digitrust.shareIdToIframe.request':
+                if(DigiTrust){
+                    DigiTrust.getUser({}, function(resp){
+                        resp.type = "Digitrust.shareIdToIframe.response";
+                        evt.source.postMessage(resp, evt.origin);
+                    }); 
+                }else{
+                    console.log("DigiTrust not found");                    
```

Comment:
Hello @goosemanjack ,
Should we keep this code part as it is or do you have something better in mind?

Reply:
There is a new logging framework I've added that will be used to replace the console.log statement. We should migrate this in a subsequent version.